suitcase/hdf5.py

```diff
@@ -96,25 +96,23 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
                     data = np.array(data)
-                    try:
+                    if data.dtype.kind in ['S', 'U']:
```

Comment:
@licode what's the name of the key for the 2D string data? I'm curious where this is coming from (probably HXN...).